sample/rules.go

```diff
@@ -125,6 +126,10 @@ func (s *RulesBasedSampler) GetSampleRate(trace *types.Trace) (rate uint, keep b
 				keep = !rule.Drop && rule.SampleRate > 0 && rand.Intn(rule.SampleRate) == 0
 				reason += rule.Name
 				s.Metrics.Histogram(s.prefix+"sample_rate", float64(rate))
+				if rule.Drop {
```

Comment:
Should we check that it's a drop before incrementing this metrics?
I was thinking this code would be in the else block from line 137